buildSrc/src/main/java/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -89,7 +90,11 @@
     @Override
     public void apply(Project project) {
         project.getRootProject().getPluginManager().apply(DockerSupportPlugin.class);
-        project.getPluginManager().apply(DistributionDownloadPlugin.class);
+        if (BuildParams.isInternal()) {
```

Comment:
We kicked the can of differentiating between internal and external builds further to here for now. 
Is this plugin even considered to be used externally. Otherwise we can remove this differentiation here and just apply the internal plugin

Reply:
I believe it is since it's needed for integration testing plugin projects.

Reply:
I was referring to `DistributionDownloadPlugin` which is the plugin that's affected by this line change, not `DistroTestPlugin`, which is simply utilizing the download plugin.

Reply:
I thought the original question was whether we needed to have internal/external behavior here, in DistroTestPlugin?

Reply:
Oh. In that case, no. But in `DistroDownloadPlugin`, yes. Sorry, I got confused.

Reply:
thank you guys for clarification.